### PR TITLE
Add OceanFacts to Math & Science

### PR DESCRIPTION
**Pull request by willuhm-js**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not have more than 100 characters
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1367,6 +1367,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Noctua](https://api.noctuasky.com/api/v1/swaggerdoc/) | REST API used to access NoctuaSky features | No | Yes | Unknown |
 | [Numbers](https://math.tools/api/numbers/) | Number of the day, random number, number facts and anything else you want to do with numbers | `apiKey` | Yes | No |
 | [Numbers](http://numbersapi.com) | Facts about numbers | No | No | No |
+| [Ocean Facts](https://oceanfacts.herokuapp.com/) | Facts pertaining to the physical science of Oceanography | No | Yes | Unknown
```
Comment (matheusfelipeog) [inline README.md:1370]: entry does not have all the required sections (have 4, need 5)

```suggestion
| [Ocean Facts](https://oceanfacts.herokuapp.com/) | Facts pertaining to the physical science of Oceanography | No | Yes | Unknown |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @willuhm-js 😄 

Can you resolve this review, please?

**Comment by willuhm-js** [comment]
> Thanks for your contribution @willuhm-js 😄 
> 
> 
> 
> Can you resolve this review, please?

Done

**Review by matheusfelipeog — APPROVED** [review]
Now it's ok @willuhm-js ;)

Thanks for the update

**Pull request merged**